features/support/helpers/wallets-helpers.js

```diff
@@ -118,12 +119,18 @@ const createWalletsSequentially = async (wallets, context) => {
   );
 };
 
-export const createWallets = async (wallets, context, sequentially) => {
+export const createWallets = async (wallets, context, options) => {
+
+  const { sequentially, skipDataLayerMigration } = options;
 
   if (sequentially === true) {
     await createWalletsSequentially(wallets, context);
   } else {
     await createWalletsAsync(wallets, context);
   }
 
+  if (skipDataLayerMigration !== true) {
```

Comment:
@daniloprates let's remove this and switch to the basic setup strategy described in my earlier comments.